packages/core/src/react-integration/hooks/hasUsableData.ts

```diff
@@ -2,11 +2,15 @@ import { FetchShape } from '@rest-hooks/core/endpoint';
 
 /** If the invalidIfStale option is set we suspend if resource has expired */
 export default function hasUsableData(
-  cacheReady: boolean,
   fetchShape: Pick<FetchShape<any>, 'options'>,
+  cacheReady: boolean,
+  notDeleted: boolean,
```

Comment:
kind of a strange boolean name - why not use `deleted`?

Reply:
yeah,  i should probably change this. initially wanted to keep true/false concept similar to cacheReady. maybe a better name can solve it so it's still true when things are 'ok'.